{{cookiecutter.project_slug}}/Makefile

```diff
@@ -63,44 +69,63 @@ fixlint: ## fix lint issues using autoflake, autopep8, and isort
 	autopep8 --in-place --recursive --aggressive tests
 	isort --apply --atomic --recursive tests
 
+.PHONY: test
 test: ## run tests quickly with the default Python
 	python -m pytest
 
+.PHONY: test-all
 test-all: ## run tests on every Python version with tox
 	tox
 
+.PHONY: coverage
 coverage: ## check code coverage quickly with the default Python
 	coverage run --source {{ cookiecutter.project_slug }} -m pytest
 	coverage report -m
 	coverage html
 	$(BROWSER) htmlcov/index.html
 
+.PHONY: clean-docs
 clean-docs: ## remove previously built docs
 	rm -f docs/{{ cookiecutter.project_slug }}.rst
 	rm -f docs/modules.rst
-	$(MAKE) -C docs clean
+	-$(MAKE) -C docs clean 2>/dev/null  # this fails if sphinx is not yet installed
 
+.PHONY: docs
 docs: clean-docs ## generate Sphinx HTML documentation, including API docs
 	sphinx-apidoc -o docs/ {{ cookiecutter.project_slug }}
 	$(MAKE) -C docs html
 	touch docs/_build/html/.nojekyll
 
+.PHONY: viewdocs
```

Comment:
Since there are hypens in all the composed names, I'd also rename this to `view-docs` and `serve-docs`